src/main/java/com/logsentinel/opendatadashboard/service/JSONStreamIterator.java

```diff
@@ -0,0 +1,80 @@
+package com.logsentinel.opendatadashboard.service;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.logsentinel.opendatadashboard.data.Record;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.Iterator;
+import java.util.zip.GZIPInputStream;
+
+
+@Service
+public class JSONStreamIterator {
+
+    private String recordsURL;
+    private Iterator<Record> streamIterator;
+    private int seek;
+
+    public JSONStreamIterator() {
+        this.recordsURL = "https://auditlog-ls.egov.bg/opendata/application/cc02fc50-5cd4-11e8-bf32-256719737274";
```

Comment:
Instead of loading it from the website, assume it is downloaded locally and just specify a path to it. We shouldn't encourage many large requests to the auditlog-ls.egov.bg server